### PR TITLE
Adding a configurable TTL cache for get_ethBalance to eth.ts

### LINKED ISSUE #1059: Add cache support to eth_getBalance
### Problem

Currently the `eth_getBalance ` queries the Mirror Node for the latest block.
Under high load a mirror node may limit RPS

### Solution

To reduce the number of calls needed and therefore the load on a mirror node a cache may be added to `eth_getBalance ` endpoint.

Mirror Node processes a block every 2 secs, however balance is updated in realtime based off of crypto transfers so a cache of 500ms may be a fair balance between optimization and data freshness.
This should be added as a configureable property

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
Adding a configurable TTL cache for get_ethBalance to eth.ts

**Related issue(s)**:

Fixes #1059 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/lib/eth.ts [hunk 1]
```diff
@@ -577,6 +578,14 @@ export class EthImpl implements Eth {
       }
     }
 
+    // check cache first
+    // create a key for the cache
+    const cacheKey = `${constants.CACHE_KEY.ETH_GET_BALANCE}-${account}-${blockNumberOrTag}`;
+    const cachedBalance = this.cache.get(cacheKey);
+    if (cachedBalance) {
+        return cachedBalance;
```
Comment (Nana-EC) [inline packages/relay/src/lib/eth.ts:586]: Add the log
```suggestion
        this.logger.trace(`${requestIdPrefix} caching ${cacheKey}:${JSON.stringify(cachedBalance)} for ${EthImpl.ethGetBalanceCacheTtlMs} ms`);
        return cachedBalance;
```

**Review by Nana-EC — COMMENTED** [review]
LG. Just add a log

**Review by david-bakin-sl — DISMISSED** [review]
OK!

File: packages/relay/src/lib/eth.ts (hunk 1 above)
Comment (AlfredoG87) [reply packages/relay/src/lib/eth.ts]: ✅  Done!

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**